script.js

```diff
@@ -0,0 +1,17 @@
+// UI
+let display = document.querySelector("calc-display");
+
+const keys = document.querySelectorAll(".number-button");
+
+let operators = document.querySelectorAll(".operation-button")
+
+// Event Handlers
+function operations(e) {
+    display.innerHTML += parseInt(e.target.innerHTML);
+
+}
+//if a reset, clear or delete
+
+// Event Listeners
+keys.forEach((e) => { e.addEventListener('click', operations)

```

Comment:
opt for event delegation rather than adding individual events in a loop to each element in your node list. 
see my example solution

Reply:
You make it look so easy. I reviewed your example solution and the DOM color switcher.

How did you get to the point of knowing what to look for in the DOM?

Especially; how do you know what to look for when using console.dir?